api/serverless.yml

```diff
@@ -0,0 +1,28 @@
+service: wetrace-api
+provider:
+  name: aws
+  runtime: python3.7
```

Comment:
not that it matters much but AWS supports python 3.8 now

Reply:
oh yeah? will bump it.